admin/proc-customizing-telemetry-destination.adoc

```diff
@@ -0,0 +1,50 @@
+[id='customizing-telemetry-destination_{context}']
+= Customizing telemetry destination
+
+
+The `analytics-provider-segment` plugin sends the collected telemetry data to {company-name} by default. However, you can configure a new destination that receives telemetry data based on your needs. For configuration, you need a unique segment key that points to the destination.
```

Comment:
1.  Let's use "Segment source" instead of "new destination".  This is the terminology Segment uses (see https://segment.com/docs/connections/find-writekey/)

2.  Fix capitalization on segment and clarify it's a write key 

`However, you can configure a new Segment source that receives telemetry data based on your needs. For configuration, you need a unique Segment write key that points to the Segment source.`


Since we are allowing customers to instrument their own Segment service to collect data, this will be the same set of data that's specified in `4. Telemetry data collection`.  Customers may need to write up their own data collection notice. Can we somehow convey to them that they can expect to see the same set data?

Reply:
@kim-tsao,  Can we mention the same in a note in this manner:
NOTE:
By configuring a new Segment source, you can collect and analyze the same set of data that is mentioned in the link:{LinkAdminGuide}#assembly-rhdh-telemetry_admin-rhdh[Telemetry data collection] section. You might also require to create your own telemetry data collection notice for your application users.